cloudflare_rules/cloudflare_httpreq_bot_high_volume_greynoise.py

```diff
@@ -0,0 +1,31 @@
+from ipaddress import ip_address
```

Comment:
Should we disable either this rule or cloudflare_rules/cloudflare_httpreq_bot_high_volume.py by default?

Reply:
Let's disable this and enable the non-greynoise version for now but we should swap that around internally